system/mavros/mavros_setup.sh

```diff
@@ -37,7 +37,7 @@ if [ "$MAVROS_TGT_SYSTEM" = "auto" ]; then
         echo "MAVROS_TGT_SYSTEM set to VEHICLE_MAVLINK_SYSID ($VEHICLE_MAVLINK_SYSID) from /etc/starling/vehicle.config"
         MAVROS_TGT_SYSTEM=$VEHICLE_MAVLINK_SYSID
     else
-        INSTANCE=$(get_instance)
+        INSTANCE=get_instance
         RESULT=$?
```

Comment:
Does this actually still call the function?
When I try a similar syntax I just get that `INSTANCE` now contains `"get_instance"` rather than the result.
I wrapped the call in `set +e` and `set -e`

Reply:
```bash
set +e
INSTANCE=$(get_instance)
set -e
```

Reply:
Sorry, can the `set -e` be moved below the `RESULT` line. I'm not sure if `set` has a return code but best to avoid it.